apps/landing/next.config.js

```diff
@@ -25,7 +25,7 @@ module.exports = withPlugins(
 		typescript: {
 			ignoreBuildErrors: true,
 		},
-		reactStrictMode: true,
+		reactStrictMode: false,
```

Comment:
I wonder why you need to change this config?

Reply:
@cloudle suggested me doing that to solve my problems about modal